### PR TITLE
Simplejson blazor

### PR DESCRIPTION
**Pull request by danielmarbach**
Alternative to #283 

Compatibility mode to v5 receiver

Left with Newtonsoft Json, right with SimpleJson

image

### CODE REVIEW

File: src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs [hunk 1]
```diff
@@ -0,0 +1,2156 @@
...
+//    limitations under the License.
+// </copyright>
+// <author>Nathan Totten (ntotten.com), Jim Zimmerman (jimzimmerman.com) and Prabir Shrestha (prabir.me)</author>
+// <website>https://github.com/facebook-csharp-sdk/simple-json</website>
+//-----------------------------------------------------------------------
+
+// VERSION:
+
+// NOTE: uncomment the following line to make SimpleJson class internal.
+#define SIMPLE_JSON_INTERNAL
```
Comment (boblangley) [inline src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs]: Should we make this all internal? Is there a reason we should make this available outside of the transport?

Comment (danielmarbach) [reply src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs]: It is internal because the define is uncommented

Comment (boblangley) [reply src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs]: Are you sure, am I reading this wrong?

https://github.com/Particular/NServiceBus.AmazonSQS/blob/3271c36541bd7c4d856f0e39169f5e376f029047/src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs#L497-L502

Comment (boblangley) [reply src/NServiceBus.AmazonSQS/SimpleJson/SimpleJson.g.cs]: Nevermind. I am reading it wrong.

**Pull request merged**